api-controllers/DashboardController.js

```diff
@@ -136,6 +141,50 @@ class DashboardController extends FabrikBaseController {
       .throw(new ContinueWithNext());
   }
 
+  validateServiceInstanceAndType(req, res) {
+    /* jshint unused:false */
+    const instance_id = req.params.instance_id;
+    const instance_type = req.params.instance_type;
+    return this._getApiServerResource(instance_id, instance_type)
+      .then(resource => _.get(resource, 'spec.options'))
+      .then(resourceOptions => {
+        const service_id = _.get(resourceOptions, 'service_id');
+        const plan_id = _.get(resourceOptions, 'plan_id');
+        const context = _.get(resourceOptions, 'context');
+        req.params.service_id = service_id;
+        req.params.plan_id = plan_id;
+        return this.fabrik.createInstance(instance_id, service_id, plan_id, context);
```

Comment:
Arent we going to get rid of director instance/ director manager classes?

Reply:
Not addressing in this PR.

Reply:
ok.